lib/rules/imports-requiring-flow.js

```diff
@@ -3,26 +3,39 @@ const path = require("path");
 const checkImport = (context, rootDir, importPath, node) => {
     const modules = context.options[0].modules || [];
 
+    const maybeReport = (importPath, testRulePath, testImportPath) => {
+        // TODO(somewhatabstract): This is not cross-platform. We need to do
+        // more work to make this rule work on Windows properly, regardless of
+        // the configuration format.
+        const subRulePath = testRulePath.endsWith(path.sep)
+            ? testRulePath
+            : `${testRulePath}${path.sep}`;
```

Comment:
I don't think this works cross-platform because I'm not sure that we're processing the incoming paths in an agnostic manner. I could be wrong.

Reply:
Also, if I understand this correctly, it feels a little weird to be appending a non-directory path with a path separate.  Not a big deal because the logic works out and it avoids check if a path is a directory.  It might be worth adding something to this comment about this.

Reply:
But it is a directory path, in reality.